cluster/vm-template-rhel7.yaml

```diff
@@ -53,4 +61,4 @@ parameters:
   value: 4096Mi
 - name: CPU_CORES
   description: Amount of cores
-  value: 4
+  value: "4"
```

Comment:
So it needs to be a string right?

Reply:
Surprisingly yes for here, the `oc process` complains about non `"` character here.
Anyway, if this default is used with the former changes above, the processed value is without quotation marks (so integer)

Reply:
Good to note: OpenShift v3.9.0-alpha.3